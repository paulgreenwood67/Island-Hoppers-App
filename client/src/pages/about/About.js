import React from "react";
import "./about.css";
import { Link } from "react-router-dom";

//function for the about page
export default function About() {
  return (
    <div className="about">
      <div className="aboutWrapper">
        <h1 className="aboutTitle">ABOUT ISLAND HOPPERS</h1>
        <h4 className="aboutDesc">
         Welcome to <span>ISLAND HOPPERS</span>.
        </h4>
        <p>
          <span>Island Hoppers</span> is a blogging platform which provides you
          the opportunity to post reviews about the amazing islands you have visited in Greece.
          We call our blogs <span> 'hops'</span> and if you join us then you will become a <span>'hopper'</span>.
          So, hop onboard and tell us about the your adventures around the beautiful islands of Greece.
        </p>
        <h2>Want to write a hop?</h2>
        <Link to="/write">
          <button>Hop over here</button>
        </Link>
      </div>
    </div>
  );
}
