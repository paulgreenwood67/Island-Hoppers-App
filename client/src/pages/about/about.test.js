//snapshot test

import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import About from './About';

test('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <About />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
