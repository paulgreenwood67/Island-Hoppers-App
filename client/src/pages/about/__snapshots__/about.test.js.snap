// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="about"
>
  <div
    className="aboutWrapper"
  >
    <h1
      className="aboutTitle"
    >
      ABOUT ISLAND HOPPERS
    </h1>
    <h4
      className="aboutDesc"
    >
      Welcome to 
      <span>
        ISLAND HOPPERS
      </span>
      .
    </h4>
    <p>
      <span>
        Island Hoppers
      </span>
       is a blogging platform which provides you the opportunity to post reviews about the amazing islands you have visited in Greece. We call our blogs 
      <span>
         'hops'
      </span>
       and if you join us then you will become a 
      <span>
        'hopper'
      </span>
      . So, hop onboard and tell us about the your adventures around the beautiful islands of Greece.
    </p>
    <h2>
      Want to write a hop?
    </h2>
    <a
      href="/write"
      onClick={[Function]}
    >
      <button>
        Hop over here
      </button>
    </a>
  </div>
</div>
`;
